test/fixtures/dhcp/dhcp2.conf

```diff
@@ -0,0 +1,3 @@
+# Test that the ISC DHCP config file parser includes files without an absolute or more specific relative path
+# If none is given, the file to be included will be looked up in the directory where the dhcp configuration file is located
+include "dhcp_subnets.conf";
```

Comment:
Please give this file nicer name like `dhcp_include_relative.conf` instead `dhcpd2.conf` which is quite confusing.